examples/chip-tool/commands/clusters/ModelCommand.h

```diff
@@ -36,22 +35,22 @@ class ModelCommand : public CHIPCommand
     void AddArguments()
     {
         AddArgument("node-id", 0, UINT64_MAX, &mNodeId);
-        AddArgument("endpoint-id", CHIP_ZCL_ENDPOINT_MIN, CHIP_ZCL_ENDPOINT_MAX, &mEndPointId);
+        AddArgument("endpoint-id", 0, UINT16_MAX, &mEndPointId);
```

Comment:
```suggestion
        AddArgument("endpoint-id", 0, UINT16_MAX - 1, &mEndPointId);
```
0xFFFF is used as a sentinel and is not a valid endpoint value.

Reply:
0xFFFF is used as a wildcard to request all endpoints.